test-suite/src/test/scala/org/scalajs/testsuite/javalib/ExpectExceptions.scala

```diff
@@ -0,0 +1,21 @@
+package org.scalajs.testsuite.javalib
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.reflect.ClassTag
+
+trait ExpectExceptions extends JasmineTest {
+
+  final def expectNoException(expr: => Any): Unit =
+    expect(expr.##).not.toThrow
```

Comment:
Oh, just because there are some helper functions that have receive an `expectThrows` which is a `=> Any` and in some cases I want to use that same helper function to test that no exception was thrown. I needed that wrapper to make the type match.